### PR TITLE
HW-52637 pay pal account second line

### PR DESCRIPTION
**Pull request by vshcherbyna-epam**

### CODE REVIEW

File: ui/src/test/java/com/hyperwallet/android/ui/transfermethod/TransferMethodUtilsTest.java [hunk 1]
```diff
@@ -134,4 +140,48 @@ public void testGetStringFontIcon_returnsDefaultValue() {
         verify(mResources, times(2)).getIdentifier(anyString(), anyString(), anyString());
         verify(mContext, times(1)).getString(anyInt());
     }
+
+    @Test
+    public void testGetSecondLine_returnsPayPalSecondLine() throws JSONException {
```
Comment (fmattos-hw) [inline ui/src/test/java/com/hyperwallet/android/ui/transfermethod/TransferMethodUtilsTest.java]: Please keep the test name consistent with the method name, given the method name getTransferMethodDetail,
the test names should follow the pattern: 
testGetTransferMethodDetail_returnsPayPalDetail
testGetTransferMethodDetail_returnsCardDetail
testGetTransferMethodDetail_returnsBankAccountDetail
testGetTransferMethodDetail_returnsPaperCheckDetail

File: ui/src/main/java/com/hyperwallet/android/ui/transfermethod/ListTransferMethodFragment.java [hunk 2]
```diff
@@ -373,16 +350,20 @@ void replaceData(List<HyperwalletTransferMethod> transferMethodList) {
 
 
             void bind(@NonNull final HyperwalletTransferMethod transferMethod) {
+                String type = transferMethod.getField(TYPE);
+                final String transferMethodIdentification = TransferMethodUtils.getTransferMethodDetail(
```
Comment (fmattos-hw) [inline ui/src/main/java/com/hyperwallet/android/ui/transfermethod/ListTransferMethodFragment.java]: please use import static as it's done in getStringResourceByName and getStringFontIcon

File: ui/src/test/java/com/hyperwallet/android/ui/transfermethod/TransferMethodUtilsTest.java (hunk 1 above)
Comment (fmattos-hw) [inline ui/src/test/java/com/hyperwallet/android/ui/transfermethod/TransferMethodUtilsTest.java]: For all those test cases, just use the builder classes to create instances of PayPal, BankAccount etc... easier than loading the json file. Just provide the data in the field which will be used to retrieve the value from. I.e 
`new HyperwalletBankAccount.Builder().bankAccountId("8017110254").build()`

File: ui/src/test/resources/paper_check_response.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (fmattos-hw) [inline ui/src/test/resources/paper_check_response.json]: Based on my previous comment, this file can be deleted

File: ui/src/test/java/com/hyperwallet/android/ui/transfermethod/TransferMethodUtilsTest.java (hunk 1 above)
Comment (vshcherbyna-epam) [reply ui/src/test/java/com/hyperwallet/android/ui/transfermethod/TransferMethodUtilsTest.java]: renamed

Comment (vshcherbyna-epam) [reply ui/src/test/java/com/hyperwallet/android/ui/transfermethod/TransferMethodUtilsTest.java]: Thanks, now it much more handy with a builder.
refactored them.

File: ui/src/test/resources/paper_check_response.json (hunk 3 above)
Comment (vshcherbyna-epam) [reply ui/src/test/resources/paper_check_response.json]: removed

File: ui/src/main/java/com/hyperwallet/android/ui/transfermethod/TransferMethodUtils.java [hunk 4]
```diff
@@ -125,4 +131,52 @@ public static String getTransferMethodName(@NonNull final Context context,
...
+     * Gets Transfer method identifier from the {@link HyperwalletTransferMethod} field
+     * by a {@link TransferMethodType}.
+     *
+     * @param context        Context
+     * @param transferMethod HyperwalletTransferMethod
+     * @param type           TransferMethodType
+     */
+    public static String getTransferMethodDetail(@NonNull Context context,
+            @NonNull final HyperwalletTransferMethod transferMethod,
+            @TransferMethodType final String type) {
```
Comment (peter-joseph) [inline ui/src/main/java/com/hyperwallet/android/ui/transfermethod/TransferMethodUtils.java]: add `@NonNull` if applicable `@Nullable` if not, but I see you checked for null after this statements so if we will know 100% probably we will have to annotate its easier to read

Comment (vshcherbyna-epam) [reply ui/src/main/java/com/hyperwallet/android/ui/transfermethod/TransferMethodUtils.java]: Added

**Pull request merged**